source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -141,6 +143,20 @@ class NiDAQmxDriverApiTests : public ::testing::Test {
     return stub()->StopTask(&context, request, &response);
   }
 
+  ::grpc::Status read_analog_f64(
+      int32 samps_per_chan,
+      uint32 array_size_in_samps,
+      ReadAnalogF64Response& response)
+  {
+    ::grpc::ClientContext context;
+    ReadAnalogF64Request request;
+    set_request_session_id(request);
+    request.set_num_samps_per_chan(100);
```

Comment:
Did you want to use the `samps_per_chan` and `array_size_in_samps` args here instead of 100 for the request fields?